sklearn/linear_model/least_angle.py

```diff
@@ -587,14 +587,15 @@ class Lars(LinearModel, RegressorMixin):
     sklearn.decomposition.sparse_encode
 
     """
+    method = 'lar'
```

Comment:
we were discussing this change before with @jnothman I suppose, that was his idea to move it here, so we'll be able to make new tests. Also he pointed that it's better to define it this way, so I'm a bit confused, honestly.

I didn't understand what l678 and l699, thought it means "line 678", but not sure what should I see on those lines then 😕